L1Trigger/Phase2L1ParticleFlow/python/l1ctJetFileWriter_cff.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
should probably be a `_cfi` instead of a `_cff` since it's defining a single module directly